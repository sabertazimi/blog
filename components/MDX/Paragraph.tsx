import { Typography } from 'antd';

interface Props {}

const Paragraph = (props: Props): JSX.Element => (
  <Typography.Paragraph {...props} />
);

export default Paragraph;
