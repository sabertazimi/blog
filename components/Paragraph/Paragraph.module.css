.p.p {
  @apply mb-9 mt-0 leading-relaxed tracking-wide;
}
