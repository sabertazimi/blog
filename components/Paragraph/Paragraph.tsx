import { Typography } from 'antd'
import cx from 'classnames'
import type { ReactNode } from 'react'
import styles from './Paragraph.module.css'

interface Props {
  children?: ReactNode
  className?: string
}

function Paragraph({ children, className, ...props }: Props): JSX.Element {
  return (
    <Typography.Paragraph
      {...props}
      className={cx(className, styles.p, 'dark:text-light')}
    >
      {children}
    </Typography.Paragraph>
  )
}

export default Paragraph
