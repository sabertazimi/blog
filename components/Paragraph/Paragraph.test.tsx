import { render } from '@testing-library/react';
import Paragraph from './Paragraph';

describe('Paragraph', () => {
  test('should render correctly (snapshot)', () => {
    const { container } = render(<Paragraph />);

    expect(container).toMatchSnapshot();
  });
});
