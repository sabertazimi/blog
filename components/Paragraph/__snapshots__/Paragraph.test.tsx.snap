// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph should render correctly (snapshot) 1`] = `
<div>
  <div
    class="ant-typography"
  />
</div>
`;
